android/src/main/kotlin/com/weplenish/flutter_wifi_connect/FlutterWifiConnectPlugin.kt

```diff
@@ -52,7 +55,7 @@ class FlutterWifiConnectPlugin() : FlutterPlugin, MethodCallHandler {
 
   override fun onMethodCall(@NonNull call: MethodCall, @NonNull result: Result) {
     when (call.method) {
-      "disconnect" && Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q -> {
```

Comment:
this was giving a compilation error

Reply:
🤦 I was testing something and forgot to remove that when checking in. I gotta get unit tests out.